dpnp/dpnp_iface_statistics.py

```diff
@@ -274,14 +275,14 @@ def cov(x1, y=None, rowvar=True, bias=False, ddof=None, fweights=None, aweights=
 
     """
 
+    if not rowvar and x1.shape[0] != 1:
```

Comment:
It needs to be moved somewhere after line#294, otherwise you can pass modified x1 in `call_origin` which has has never be done. We shall pass arguments in numpy fallback function in the same way we received them in dpnp.cov() call, without any modification.
This is the reason why some tests failed in internal CI loop for the commit.